lib/db2.js

```diff
@@ -76,6 +76,8 @@ DB2.prototype.create = function(model, data, options, callback) {
  * @param {Function} cb The callback function
  */
 DB2.prototype.update = function(model, where, data, options, cb) {
+  debug('DB2.prototype.update (enter): where=%j, data=%j, options=%j\n',
```

Comment:
Nitpick: I do not think if `(enter)` is really needed; when you get `DB2.prototype.update` it is obvious it is already entered in this section